docker-compose/run_selenium_tests_local_keybase.sh

```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+
+# Run the selenium tests in docker
+# 
+# NOTE:
+#
+#   1. You must be logged in to Keybase and have the BB2 team file system mounted.
+#
+#   2. You must also be connected to the VPN.
+#
+# SETTINGS:  You may need to customize these for your local setup.
+
+HOSTNAME_URL="http://bb2:8000"
+
+KEYBASE_ENV_FILE="team/bb20/infrastructure/creds/ENV_secrets_for_local_integration_tests.env"
+KEYBASE_CERTFILES_SUBPATH="team/bb20/infrastructure/certs/local_integration_tests/fhir_client/certstore/"
+
+BB20_TEAM_VAULT_PASSFILE="/cygdrive/k/team/bb20/infrastructure/creds/vault_pw.txt"
```

Comment:
This appears to be something else that breaks on Mac/Linux

Reply:
Per below comment, the ansible vault dependency should be removed entirely for .env file secrets from keybase